NitroxServer/GameLogic/EventTriggerer.cs

```diff
@@ -4,19 +4,22 @@
 using System.Timers;
 using NitroxModel.Logger;
 using NitroxModel.Packets;
-using NitroxServer.GameLogic.Bases;
 
 namespace NitroxServer.GameLogic
 {
     public class EventTriggerer
     {
-        private PlayerManager playerManager;
+        private readonly Dictionary<string, Timer> eventTimers;
+        private readonly PlayerManager playerManager;
         private Stopwatch stopWatch;
-        private Dictionary<String, Timer> eventTimers = new Dictionary<string, Timer>();
+        
+
         public double ElapsedTime;
         public double AuroraExplosionTime;
+
         public EventTriggerer(PlayerManager playerManager, double elapsedTime, double? auroraExplosionTime)
         {
+            eventTimers = new();
```

Comment:
Minor: I personally would move this back to the field as it has no need to be in the constructor, but do as you think.